Rakefile

```diff
@@ -28,6 +24,74 @@ namespace :yard do
     sh "bundle exec yard graph --protected --full --dependencies > #{output_file}"
     puts 'open doc/erd.dot if you have graphviz installed'
   end
+
+  # #files => Array<String>
+  # The Ruby source files (and any extra documentation files separated by '-') to process.
+  DOC_FILES = ['lib/**/*.rb', '-', 'README.md']
+  DOC_FILES.concat Rake::FileList['docs/**/*.md'].to_a
```

Comment:
still building these to wrong relative url, e.g. https://rails-api.github.io/active_model_serializers/file.configuration_options.html instead of in docs